runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsPlugin.java

```diff
@@ -62,4 +81,34 @@ protected void shutDown() throws Exception
 	{
 		overlayManager.remove(overlay);
 	}
+
+	@Subscribe
+	public void onGameTick(GameTick e)
+	{
+		final WorldPoint playerPos = client.getLocalPlayer().getWorldLocation();
+
+		if (lastPlayerPosition != null)
+		{
+			if (!playerPos.equals(lastPlayerPosition))
+			{
+				lastTickPlayerMoved = client.getTickCount();
+			}
+			else if (lastTickPlayerMoved + 1 == client.getTickCount())
+			{
+				lastTimePlayerStoppedMoving = System.currentTimeMillis();
+			}
+		}
+
+		lastPlayerPosition = playerPos;
+	}
+
+	@Subscribe
+	public void onGameStateChanged(GameStateChanged e)
+	{
+		if (e.getGameState() == GameState.LOGIN_SCREEN)
+		{
+			// When true tile fadeout is enabled, this prevents the true tile from showing up right after logging in.
+			lastPlayerPosition = null;
```

Comment:
This should also either be cleared on shutdown, or set on startup. As-is, you can toggle the plugin off, move a few tiles, then turn it back on to see up to 1 tick of it displaying in the wrong location.

Reply:
I decided to clear it on shutdown.